README.md

```diff
@@ -49,33 +49,38 @@ On the endpoint, JSON will be POSTed with the following structure:
+| JWT_SECRET_PATH                               | No       | /data/neth/jwt.hex    | Path to JWT secret hex file for Nethermind RPC authentication. If provided, will generate a new JWT token for each execution node request                                                                                                               |
+| LOCATOR_ADDRESS                               | Yes      | 0x123                 | Address of the Locator contract [Goerli](https://docs.lido.fi/deployed-contracts/goerli/) / [Mainnet](https://docs.lido.fi/deployed-contracts/)                                                                                                         |
+| STAKING_MODULE_ID                             | Yes      | 123                   | Staking Module ID for which operator ID is set, currently only one exists - ([NodeOperatorsRegistry](https://github.com/lidofinance/lido-dao#contracts)) with id `1`                                                                                    |
+| OPERATOR_ID                                   | Yes      | 123                   | Operator ID in the Node Operators registry, easiest to get from Operators UI: [Goerli](https://operators.testnet.fi)/[Mainnet](https://operators.lido.fi)                                                                                               |
+| OPERATOR_IDENTIFIERS                          | No       | [0,1,2]               | Alternative to OPERATOR_ID. Array of Operator IDs to handle exits for multiple operators simultaneously                                                                                                                                                 |
+| MESSAGES_LOCATION                             | No       | messages              | Local folder or external storage bucket url to load json exit message files from. Required if you are using exit messages mode                                                                                                                          |
+| VALIDATOR_EXIT_WEBHOOK                        | No       | http://webhook        | POST validator info to an endpoint instead of sending out an exit message in order to initiate an exit. Required if you are using webhook mode                                                                                                          |
+| ORACLE_ADDRESSES_ALLOWLIST                    | Yes      | ["0x123"]             | Allowed Oracle addresses to accept transactions from [Goerli](https://testnet.testnet.fi/#/lido-testnet-prater/0x24d8451bc07e7af4ba94f69acdd9ad3c6579d9fb/) / [Mainnet](https://mainnet.lido.fi/#/lido-dao/0x442af784a788a5bd6f42a01ebe9f287a871243fb/) |
```

Comment:
- broken link (https://testnet.testnet)
- let's update the Goerli part here and replace it with the actual locator from Hoodi